weTranslate/FavoriteCategoriesTableViewCell.swift

```diff
@@ -0,0 +1,120 @@
+                backgroundImageView.image = UIImage(named: viewModel.background)
+            }
+        }
+    }
+
+    private let overlayView: UIView = {
+        let view = UIView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .blackColor()
+        view.layer.opacity = 0.2
+        return view
+    }()
+
+    private let backgroundImageView: UIImageView = {
+        let imageView = UIImageView(frame: .zero)
+        return imageView
+    }()
+
+    private let wordsCountOverlayView: UIView = {
+        let view = UIView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .blackColor()
+        view.layer.opacity = 0.6
+        view.layoutMargins = UIEdgeInsets(top: 5, left: 10, bottom: 5, right: 10)
+        view.layer.cornerRadius = 5
+        view.layer.masksToBounds = true
+        return view
+    }()
+
+    private let wordsCountLabel: Label = {
+        let label = Label(textColor: .whiteColor(), font: Font.font(style: .Subheadline))
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private let languageLabel: Label = {
+        let label = Label(textColor: .whiteColor(), font: Font.font(style: .Title1, weight: .Medium))
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private let columnView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .Center
+        stackView.spacing = 10
+        return stackView
+    }()
+
+    private let bodyView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .Vertical
+        stackView.alignment = .Center
+        stackView.spacing = 5
+        return stackView
+    }()
+
+
+    // MARK: - Init
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.opaque = true
+        backgroundColor = Color.background
+        opaque = true
+
+        wordsCountOverlayView.addSubview(wordsCountLabel)
+
+        bodyView.addArrangedSubview(languageLabel)
+        bodyView.addArrangedSubview(wordsCountOverlayView)
+        columnView.addArrangedSubview(bodyView)
+
+        contentView.addSubview(overlayView)
+        contentView.addSubview(columnView)
+        backgroundView = backgroundImageView
+
+        let margins = contentView.layoutMarginsGuide
+        columnView.leadingAnchor.constraintEqualToAnchor(margins.leadingAnchor).active = true
+        columnView.trailingAnchor.constraintEqualToAnchor(margins.trailingAnchor).active = true
+        columnView.topAnchor.constraintEqualToAnchor(margins.topAnchor).active = true
+        columnView.bottomAnchor.constraintEqualToAnchor(margins.bottomAnchor).active = true
```

Comment:
We might want a category on UIView that take a view (layoutGuide or view) and apply those constraints.